packages/material-ui-lab/src/internal/svg-icons/createSvgIcon.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+export default function createSvgIcon(path, displayName) {
```

Comment:
What's the difference to the utility from core? Should we make it public?

Reply:
It's the same utility. We could